services/ui-src/src/components/reports/ModalOverlayReportPage.tsx

```diff
@@ -56,6 +57,10 @@ export const ModalOverlayReportPage = ({
     useStore().user ?? {};
   const { report } = useStore();
 
+  const { MLR: accordionMLR, MCPAR: accordionMCPAR } = accordionVerbiage;
+  const accordionReport =
+    report?.reportType === ReportType.MCPAR ? accordionMCPAR : accordionMLR;
```

Comment:
this component is only used in MLR for programs and now in MCPAR for quality measures